src/components/Filters/components/Number/Number.tsx

```diff
@@ -24,7 +30,7 @@ const Number: React.FC<NumberProps> = ({ value = '', onChange, min, max }) => {
     <StyledNumber
       max={max}
       min={min}
-      placeholder="Number"
+      placeholder={placeholder}
       step="any"
```

Comment:
We probably should be able to set custom `step` value

Reply:
We probably should but we also should display step buttons which are hidden now. It is design requirement. The change would be rather small. I will discuss it with UX and eventually get back to it under separate ticket.